src/users/06-fediverse-interaction.md

```diff
@@ -0,0 +1,58 @@
+# Interacting with Lemmy from the rest of the fediverse
+
+As mentioned in the introduction, Lemmy users and non-Lemmy users from the
+fediverse can communicate between each other, too. The way Lemmy users' activity
+is handled by non-Lemmy instances depends on the software running on the latter.
```

Comment:
Don't place these fake code newlines. Just write markdown in proper paragraphs. You might need to use a markdown editor if your current code editor is inserting these needlessly.